assets/app/features/upgrade/controllers/upgrade-nodes.controller.js

```diff
@@ -40,6 +46,12 @@
             completed: false,
             running: false,
             spinnerVisible: false,
+            upgradeComputeCheckboxSelected: null,
```

Comment:
I would rename it to e.g. `computeUpgradeEnabled`. Models should not refer to view terms.

Reply:
will do